kernel/Kconfig

```diff
@@ -464,6 +464,14 @@ config SCHED_THREAD_USAGE
 	help
 	  Collect thread runtime info at context switch time
 
+config THREAD_RUNTIME_STATS_BETWEEN_COLLECTIONS
+	bool "Keep track of thread runtime statistics between collections"
+	default n
+	depends on SCHED_THREAD_USAGE
+	help
+	  Enable collection of runtime statistics between metrics collections.
```

Comment:
Wording here is awkward, enable collection between collections?
Same for the Kconfig itself.